.env

```diff
@@ -16,7 +16,16 @@ HTTPS_PORT=443
 # Docker
 DOCKER_REGISTRY_URL=docker.sdelements.com:443/moodle
 
+# Certbot variables
+CERT_DOMAIN=localhost
+CERT_DIR=/etc/nginx/ssl/certbot
+CHALLENGE_DIR=/var/letsencrypt/challenges
+ADMIN_EMAIL=devops-admin@securitycompass.com
+
 # Moodle variables
+# WWWROOT MUST be IP or FQDN
+MOODLE_WWWROOT=https://${CERT_DOMAIN}
+MOODLE_DATAROOT=/opt/moodle/moodledata
```

Comment:
I think you copied this line by accident.. duplicated below..

Reply:
Fixed.

Reply:
`MOODLE_DATAROOT=/opt/moodle/moodledata` is also duplicate